truss/cli/resolvers/model_team_resolver.py

```diff
@@ -0,0 +1,79 @@
+"""Team resolution logic for models."""
+
+from typing import Optional
+
+import click
+
+from truss.cli import remote_cli
+from truss.remote.baseten.core import get_model_team_id
+from truss.remote.baseten.remote import BasetenRemote
+
+
+def resolve_model_team_name(
```

Comment:
nit: would be cool to generalize this logic across chains, training projects, and models if its possible